_posts/2023-01-31-datacater-uses-quarkus-to-make-data-streaming-accessible.adoc

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: 'DataCater uses Quarkus to make Data Streaming more accessible'
+date: 2023-01-31
```

Comment:
Could you change the date to Thursday here and in the file name. We will publish the article on Thursday and tweet about it.

Reply:
I updated the date to Thursday, February 9. ✅